gate/gate-web/src/test/groovy/com/netflix/spinnaker/gate/FunctionalSpec.groovy

```diff
@@ -119,7 +119,7 @@ class FunctionalSpec extends Specification {
 
     def localPort = ctx.environment.getProperty("local.server.port")
     api = new Retrofit.Builder()
-        .baseUrl("http://localhost:${localPort}")
+        .baseUrl("http://localhost:${localPort}/")
```

Comment:
Omit the trailing slash and use RetrofitUtils.getBaseUrl here?

Reply:
Done